### DISCUSSION TITLE
Should we add NoSuchElementExceptionMapper as a "standard" mapper?

### DISCUSSION THREAD

**Discussion by sleberknight** [kiwiproject/dropwizard-service-utilities#269] [General]
We recently [added](https://github.com/kiwiproject/kiwi/issues/748) the `NoSuchElementExceptionMapper` to kiwi, which maps `NoSuchElementException` to 404 responses. Should this be registered as a "standard" exception mapper in `StandardExceptionMappers`?

**Comment by chrisrohr** [comment]
Yes I think so.

On Tue, Jul 19, 2022 at 9:57 PM Scott Leberknight ***@***.***>
wrote:

> We recently added <https://github.com/kiwiproject/kiwi/issues/748> the
> NoSuchElementExceptionMapper to kiwi, which maps NoSuchElementException
> to 404 responses. Should this be registered as a "standard" exception
> mapper in StandardExceptionMappers?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kiwiproject/dropwizard-service-utilities/discussions/269>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAMVUGFS2MELBUHC6Q3G4DVU5MJ5ANCNFSM54B4U7PA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>
-- 
Thanks,
Chris

**Comment by sleberknight** [comment]
Fixed by #424
